examples/gpu/example_profile.py

```diff
@@ -0,0 +1,15 @@
+#Script to show how to run. 
```

Comment:
How to run what? You mean `new_analyzer.py` but imagine I'm a user and I just stumble on this file.